capybara-webkit.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 1.9.0"
 
-  s.add_runtime_dependency("capybara", ">= 2.3.0", "< 2.5.0")
+  s.add_runtime_dependency("capybara", ">= 2.3.0")
```

Comment:
Capybara tries to use semantic versioning, so by definition Capybara 3.0 will have breaking changes.  I can also pretty much guarantee you this change won't be accepted with an open ended version (although I have no say on it)